tools/importPlanGoals.test.js

```diff
@@ -7,6 +8,39 @@ describe('Import TTA plan goals', () => {
   afterAll(() => {
     db.sequelize.close();
   });
+  describe('Update grants and grantees', () => {
```

Comment:
I like these tests, but I'm confused why they're in this file rather than one that matches the filename where the source is.

Reply:
Good observation. Originally, I had them in a separate file, but due to `jest `running tests in parallel, I was getting inconsistent results. Perhaps we can postpone separating these for later to investigate exactly why the discrepancies. I didn't think spending more time on this now was worth it.

Reply:
My guess is that it's an impact of the mix of explicit ids and auto increment.